catascript/.env.template

```
@@ -0,0 +1,6 @@
+DATA_ROOT = /path/to/data_catalog
+PATH_TO_EXTRACTED_TICS = /path/to/extracted/TICS/file/containing/dictionnary.pickle
+LIST_DB_FIELDS = list,of,fields,in,this,format,and,in,the,order,they,appear,in,catalog
+OTHER_MISSIONS_IDS = sublist,from,listdb_fields,containing,ids,from,other,missions
+ENGINE_URL = dialect+driver://username:password@host:port/database
```

The syntax in the actual `.env` file seems to be more like `dialect+driver://username:password@host/database`
```suggestion
ENGINE_URL = dialect+driver://username:password@host/database
```